aws-s3/authorization.mli

```diff
@@ -47,4 +47,16 @@ val chunk_signature:
   scope:string ->
   previous_signature:string ->
   sha:Digestif.SHA256.t -> Digestif.SHA256.t
+
+val make_presigned_url :
```

Comment:
Maybe include the signature in the generated documentation and add some comments of how and for what its used?

Reply:
I tried to do this, I'm not sure I did it exactly correct so let me know if you have any other comments.

Reply:
that was a good suggestion, thanks.